### ISSUE TITLE
Some foreign characters are not rendered properly

### ISSUE THREAD

**Issue by shankar2k** [rember/anki-wrapped#1]
![Anki Wrapped 2024](https://github.com/user-attachments/assets/fae14b42-82e8-41ec-b944-697f9224654a)

I am using Anki to learn Slovak from books.  The character 'ň' and 'ž' were not rendered correctly, though they work fine in Anki. Maybe the text should use a font with more unicode support?

**Comment by giacomoran** [comment]
Yeah, maybe it's sufficient to set a locale in satori https://github.com/vercel/satori?tab=readme-ov-file#locales
